upgrade/helpers/tasks.py

```diff
@@ -49,19 +70,51 @@ class ProductNotFound(Exception):
     """Raise if the product you are searching is not found"""
 
 
+def save_server_config(satellite_host):
+    """
+    Save the satellite host details in the nailgun server config, that helps to execute
+    all the nailgun API's
+    :param satellite_host: The satelite hostname
+    :return:
+    """
+    sat_url = f'https://{satellite_host}'
+    ServerConfig(url=sat_url, auth=['admin', 'changeme'], verify=False).save()
+
+
+def capsule_details_update(capsules):
+    """
+    This function used to update the required details of the capsule.
+    :param capsules:
+    """
+    for capsule in capsules:
+        smart_proxy = (
+            entities.SmartProxy()
+            .search(query={'search': f'name={capsule}'})[0]
+            .read()
+        )
+        loc = entities.Location().search(query={'search': 'name="Default Location"'})[0]
+        org = entities.Organization().search(query={'search': 'name="Default Organization"'})[0]
+        smart_proxy.location.append(entities.Location(id=loc.id))
+        smart_proxy.update(['location'])
+        smart_proxy.organization.append(entities.Organization(id=org.id))
+        smart_proxy.update(['organization'])
+        with fabric_settings(warn_only=True):
+            run(f"hammer capsule content add-lifecycle-environment --organization-id "
```

Comment:
Cant we use nailgun here ?

Reply:
I tried but the changes don't reflect through nailgun, that's why I used the hammer command.